Documentation/issue-guide.md

```diff
@@ -17,7 +17,7 @@ We use GitHub labels on our issues in order to classify them.  We have the follo
  * [test bug](https://github.com/dotnet/wcf/labels/test%20bug) is a label we assign to all issues related to testing. We label these further with "infrastructure" (for test infrastructure changes) or "enhancement" (for missing tests or those that need enhancement).
 
 **Ownership**: These labels are used to specify who owns specific issue. Issues without an ownership tag are still considered "up for discussion" and haven't been approved yet. We have the following different types of ownership:
- * [up for grabs](https://github.com/dotnet/wcf/labels/up%20for%20grabs): Small sections of work which we believe are well scoped. These sorts of issues are a good place to start if you are new.  Anyone is free to work on these issues.
+ * [up for grabs](https://github.com/dotnet/wcf/labels/up-for-grabs): Small sections of work which we believe are well scoped. These sorts of issues are a good place to start if you are new.  Anyone is free to work on these issues.
```

Comment:
Change the visible name also to "up-for-grabs" (replace spaces with -) to provide unified clarity.